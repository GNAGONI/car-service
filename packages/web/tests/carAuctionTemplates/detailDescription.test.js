import React from 'react'
import renderer from 'react-test-renderer'
import DetailDescription from '../../components/carAuctionTemplates/detailDescription'

it('renders correctly', () => {
  const tree = renderer.create(<DetailDescription />).toJSON()
  expect(tree).toMatchSnapshot()
})
