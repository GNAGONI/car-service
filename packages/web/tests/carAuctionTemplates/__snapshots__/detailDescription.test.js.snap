// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="detail-description info-box"
>
  <ul
    className="description-list"
  >
    <li>
      <span
        className="detail-title"
      >
        Bodywork condition
      </span>
      <p>
        
      </p>
    </li>
    <li>
      <span
        className="detail-title"
      >
        Interior condition
      </span>
      <p>
        
      </p>
    </li>
    <li>
      <span
        className="detail-title"
      >
        Exterior
      </span>
      <p>
        
      </p>
    </li>
  </ul>
  <ul
    className="description-list"
  >
    <li>
      <span
        className="detail-title"
      >
        Interior
      </span>
      <p>
        
      </p>
    </li>
    <li>
      <span
        className="detail-title"
      >
        Safety features
      </span>
      <p>
        
      </p>
    </li>
  </ul>
</div>
`;
